sdk/python/feast/infra/offline_stores/offline_store.py

```diff
@@ -179,6 +179,19 @@ def pull_latest_from_table_or_query(
         end_date: datetime,
     ) -> RetrievalJob:
         """
+        This method is invoked when running materialization (using the feast materialize or feast
```

Comment:
`Feast`, not `feast`

Reply:
It is part of a command, added backticks to make more clear.